OpenRA.Mods.Common/Traits/Render/WithAttackAnimation.cs

```diff
@@ -100,7 +109,10 @@ void ITick.Tick(Actor self)
 			if (!string.IsNullOrEmpty(prefix) && sequence != (prefix + sequence))
 				sequence = prefix + sequence;
 
-			wsb.DefaultAnimation.ReplaceAnim(sequence);
+			if (wst == null)
```

Comment:
`wst` and `wsb` are different classes (they don't share a parent), that just happen to have separate methods with the same name that do the same thing.  This therefore won't work.